src/endpoints.ts

```diff
@@ -0,0 +1,25 @@
+import * as https from 'https';
```

Comment:
@SimonSchick @connor4312 it adds 50kb to our browser build of this. Webpacked from inside this repo.

Im not sure how FE2 webpack would treat it though.